pkg/asset/config/appliance_config.go

```diff
@@ -248,20 +255,33 @@ func (a *ApplianceConfig) validateConfig() field.ErrorList {
 }
 
 func (a *ApplianceConfig) validateImageRegistry() field.ErrorList {
+	allErrs := field.ErrorList{}
+
 	if a.Config.ImageRegistry == nil {
 		return nil
 	}
 
-	// TODO(MGMT-15163): add validation for a.Config.ImageRegistry.URI
+	if a.Config.ImageRegistry.URI != nil {
+		cmd := fmt.Sprintf(PodmanInspect, swag.StringValue(a.Config.ImageRegistry.URI))
+		logrus.Debugf("Running uri validation cmd: %s", cmd)
+		args := strings.Split(cmd, " ")
```

Comment:
not relevant for this PR, but we might want to refactor this into some helper func. or maybe add an 'Execute' interface that gets a cmd and do the split internally:/

Reply:
I agree.